src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ CHANGELOG
  * preferred choices are repeated in the list of all choices
  * deprecated using `int` or `float` as data for the `NumberType` when the `input` option is set to `string`
  * The type guesser guesses the HTML accept attribute when a mime type is configured in the File or Image constraint.
+ * Added `getIsEmptyCallback()` method to `FormConfigInterface`.
+ * Added `setIsEmptyCallback()` method to `FormConfigBuilderInterface`.
```

Comment:
lowercase first letter as the other items

Reply:
What is the exact rule for this? Sometimes it's lowercased without final point, sometimes not. Shouldn't we normalize a format and instore a rule?